src/index.tsx

```diff
@@ -58,7 +58,7 @@ const propsMap: { [key: string]: InputProps } = {
   }
 };
 
-const AuthCode = forwardRef<AuthCodeRef, Props>(
+export const AuthCode = forwardRef<AuthCodeRef, AuthCodeProps>(
```

Comment:
https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/

Also makes VSCode auto-import more precise.

Reply:
Hi @heldr I just merged #49 with the component props exported, for doing a named export instead of default I will need to take a closer look, so I didn't make the change in this PR